samples/ipc/ipc_service/sample.yaml

```diff
@@ -57,6 +57,13 @@ tests:
       - nrf5340dk/nrf5340/cpuapp
     extra_args: >
       FILE_SUFFIX=icmsg
+  sample.ipc.ipc_service.nrf5340dk_icbmsg_default:
+    platform_allow:
+      - nrf5340dk/nrf5340/cpuapp
+    integration_platforms:
+      - nrf5340dk/nrf5340/cpuapp
+    extra_args: >-
```

Comment:
`-` is off

Reply:
It truncates the newline at the end of FILE_SUFFIX.
Without it we got a warning:
```
CMake Warning:
  Value of FILE_SUFFIX contained a newline; truncating
```
This does not appear in other positions as there are other extra arguments after FILE_SUFFIX. Other solution would be to use a list:
```
 sample.ipc.ipc_service.nrf5340dk_icbmsg_default:
    platform_allow:
      - nrf5340dk/nrf5340/cpuapp
    integration_platforms:
      - nrf5340dk/nrf5340/cpuapp
    extra_args:
      - FILE_SUFFIX=icbmsg
```

Reply:
Changed all extra_args to lists.